### DISCUSSION TITLE
SCInsta downloader not working

### DISCUSSION THREAD

**Discussion by krislicoque** [SoCuul/SCInsta#115] [Support] [answered]
When trying to save an image I get this, and no option to download:

![img_6923](https://github.com/user-attachments/assets/8504d886-b5e2-4efc-8a0d-7f843498e2ed)

Furthermore saving videos doesn't even give the prompt.

**Comment by SoCuul** [comment, marked as answer]
As mentioned in the readme, the media downloading features are unfortunately not currently working. I'm hopeful to revamp them in a future version.
